dev/com.ibm.ws.concurrent_fat/publish/servers/concurrent.spec.fat/server.xml

```diff
@@ -10,21 +10,20 @@
   <include location="../fatTestPorts.xml"/>
   <variable name="onError" value="FAIL"/>
 
-  <!-- TODO remove the internal attribute (and overrides of default instances) once managed executors are switched over to using policy executor -->
-  <managedExecutorService id="DefaultManagedExecutorService" policyExecutor.internal.prototype.do.not.use="enabled-for-internal-testing-only"/>
-  <managedScheduledExecutorService id="DefaultManagedScheduledExecutorService" policyExecutor.internal.prototype.do.not.use="enabled-for-internal-testing-only"/>
+  <managedExecutorService id="DefaultManagedExecutorService"/>
+  <managedScheduledExecutorService id="DefaultManagedScheduledExecutorService"/>
```

Comment:
Remove the overrides of default instances here too.  The overrides were only added to temporarily include the internal property